app/models/comments-model.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../../db/connection')
+
+exports.fetchCommentsById = (id) => {
+    return db.query(`SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id FROM comments INNER JOIN articles ON comments.article_id = articles.article_id WHERE comments.article_id = ${id} ORDER BY created_at ASC;`)
```

Comment:
You've over complicated this a little, you're querying the comments table by article id. No need for a join. 

Your Promise reject may be firing incorrectly, what about a valid article id with no comments? What status code would that be and why?